controller/array_action/messages.py

```diff
@@ -17,6 +17,8 @@
 
 PoolDoesNotExist_message = "Pool does not exist: {0} , array : {1}"
 
+PoolIsRequired = "pool parameter is mandatory in {0}"
```

Comment:
sentences convention is to begin with Upper case

Reply:
done